Gemfile.lock

```diff
@@ -102,4 +102,4 @@ DEPENDENCIES
   xctasks (~> 0.5.0)
 
 BUNDLED WITH
-   1.10.6
+   1.12.4
```

Comment:
I reverted these changes in the Lock file and Tests now run gracefully.
New Test is called „testPaginatorWithPaginationURLThatIncludesAColon“ and also succeeds. 





> Am 24.05.2016 um 13:35 schrieb Valerio Mazzeo <notifications@github.com>:
> 
> In Gemfile.lock <https://github.com/RestKit/RestKit/pull/2427#discussion_r64374471>:
> 
> > @@ -102,4 +102,4 @@ DEPENDENCIES
> >    xctasks (~> 0.5.0)
> >  
> >  BUNDLED WITH
> > -   1.10.6
> > +   1.12.4
> can you please revert this changes?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/RestKit/RestKit/pull/2427/files/4e68b3b5035ad6566ec5314582e213ab5382f191#r64374471>